scvi/inference/posterior.py

```diff
@@ -871,7 +889,13 @@ def differential_expression_score(
         :param all_stats: whether additional metrics should be provided
         :\**kwargs: Other keywords arguments for `get_sample_scale()`
 
-        :return: Differential expression properties
+        :return: Differential expression properties. The most important keys are:
+
+            - proba_de (probability of being differentially expressed in change mode)
```

Comment:
let's put wrap the names with ``name`` to make it look nice in RTD. It's two of ` on each side.